### ISSUE TITLE
`<<< "..."` causes the entire file to be treated as string

### ISSUE THREAD

**Issue by martin-braun** [tree-sitter/tree-sitter-bash#148]
The command `cat /dev/stdin <<< "hello world"` causes the entire file to be treated as string:

<img width="278" alt="Screen Shot 2023-02-16 at 3 32 43 PM" src="https://user-images.githubusercontent.com/24259245/219393554-b21cdf80-234c-4232-89f8-e4d942fb9030.png">

`<<<` is used to pipe in direct text. This is useful when regular piping isn't working, i.e. on macOS when trying to source a downloaded script and passing arguments to it: 

```sh
wexec() {
	url="$1"
	shift
	if command -v "curl" > /dev/null 2>&1; then
		\. /dev/stdin <<< "$(curl --proto '=https' --tlsv1.2 -s "$url")"
	elif command -v "wget" > /dev/null 2>&1; then
		\. /dev/stdin <<< "$(wget --secure-protocol=TLSv1_2 -qO- "$url")"
	else
		echo "No curl or wget found." >&2
		return 1
	fi
}
```

<img width="585" alt="Screen Shot 2023-02-16 at 3 34 55 PM" src="https://user-images.githubusercontent.com/24259245/219393987-d6e34eb5-cec8-44f6-a1f2-d9bef7915a19.png">

This is probably related to #140, but it was not fixed for the `<<<` case.

**Comment by verhovsky** [comment]
I typed your code 

```sh
cat /dev/stdin <<< "hello world"
# test
echo test
```

into https://tree-sitter.github.io/tree-sitter/playground and it parses it correctly

```
program [0, 0] - [3, 0]
  redirected_statement [0, 0] - [0, 32]
    body: command [0, 0] - [0, 14]
      name: command_name [0, 0] - [0, 3]
        word [0, 0] - [0, 3]
      argument: word [0, 4] - [0, 14]
    redirect: herestring_redirect [0, 15] - [0, 32]
      string [0, 19] - [0, 32]
  comment [1, 0] - [1, 6]
  command [2, 0] - [2, 9]
    name: command_name [2, 0] - [2, 4]
      word [2, 0] - [2, 4]
    argument: word [2, 5] - [2, 9]
```

**Comment by martin-braun** [comment]
@verhovsky Does the playground use the version 0.20.7 or bleeding edge? #140 was fixed in December, but the last release was in September as I just recognized. It could be that my issue is already covered by #140, but there was no update yet.

**Comment by verhovsky** [comment]
Looks like it clones the master branch

https://github.com/tree-sitter/tree-sitter/blob/c51896d32dcc11a38e41f36e3deb1a6a9c4f4b14/script/fetch-fixtures#L24

**Comment by martin-braun** [comment]
@verhovsky Since the playground confirms it is fixed in master, I assume it was fixed with #140. When is the next release planned btw?

Thanks, I will close this and re-open if there are problems with the next release in that regard.

**Comment by martin-braun** [comment]
Thanks, I'm using coc.nvim for LSP stuff, though. I will wait for the release, it's not a blocker. @skovhus
